src/Spryker/Zed/UuidBehavior/Persistence/Propel/Behavior/templates/objectUpdateUuidBeforeUpdate.php

```diff
@@ -0,0 +1,10 @@
+
+/**
+ * @return void
+ */
+protected function updateUuidBeforeUpdate()
+{
+    if (empty($this->getUuid())) {
```

Comment:
Please do not use empty with methods. Use just

    if (!$this->getUuid()) 

etc